docs/topics/mta-review-reports.adoc

```diff
@@ -152,6 +155,56 @@ image::3-5-expanded-issue.png[Expanded rule in the issues report]
 
 Issues are sorted into four categories by default. Information on these categories is available at ask Category.
 
+
+=== Insights
+
+Insights provide helpful information about the technologies used in the application and their usage in the code.
+For example, a rule searching for a deprecated API usage in the code that does not impact the current migration but can be tracked and fixed when needed in the future.
+
+Unlike issues, insights do not need to be fixed for a successful migration. They are generated by any tagging or non-tagging rule and do not have an effort and category assigned. They might have a message and tag.
+
+[NOTE]
+=====

```

Comment:
Wording is a bit odd to say that it's just available in the CLI for the moment.

Reply:
I would reword to be "At this time, application insights are viewable in CLI generated analysis reports at this time" or something like that...

Reply:
reworded.